requirements.txt

```diff
@@ -2,4 +2,5 @@ requests>=2.12.4            # api tests
 selenium>=2.53.6            # web tests
 Appium-Python-Client>=0.24  # mobile tests
 six>=1.10.0
+behave==1.2.5
```

Comment:
Why  the 1.2.5 version? 1.2.6 or 1.2.7 are available.

Reply:
upgraded to `behave==1.2.6` (last published release: https://pypi.org/project/behave/)